components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -17,6 +17,8 @@
 
 namespace brave_wallet {
 
+constexpr char kBraveServicesKeyHeader[] = "x-brave-key";
```

Comment:
Looks like we have another `kBraveServicesKeyHeader` outside `brave_wallet` namespace with a different value.

Started a discussion on Slack ([thread](https://bravesoftware.slack.com/archives/C7VLGSR55/p1683552517066739)) about using a single header for all services. This change still makes sense because `x-brave-key` is mostly used for wallet stuff. I can harmonise all the header keys in different PR once there's agreement with devops.

Reply:
SGTM to leave this out side of scope for this work.